.claude/skills/syncing-atuin/references/troubleshooting.md

```
@@ -469,15 +467,15 @@ programs.atuin.settings = {
 
 > **발생 시점**: 2026-03-27
 > **환경**: atuin 18.12.1 (nixpkgs) + DB에 18.13.0 migration 적용됨
-> **상태**: 해결 (nixpkgs 업데이트, PR #333)
+> **상태**: 해결 (nixpkgs 업데이트)
 
 **증상**: `atuin history list` 등 모든 히스토리 관련 명령이 실패.
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)